third_party/move/move-model/src/metadata.rs

```diff
@@ -17,8 +17,8 @@ use std::{
 };
 
 const UNSTABLE_MARKER: &str = "-unstable";
-pub const LATEST_STABLE_LANGUAGE_VERSION: &str = "2.1";
-pub const LATEST_STABLE_COMPILER_VERSION: &str = "2.0";
+pub const LATEST_STABLE_LANGUAGE_VERSION: LanguageVersion = LanguageVersion::V2_1;
```

Comment:
This code does not compile (try running `cargo build` from the aptos-core root directory or take a look at the failing CI).

Reply:
oops, checked tests only, forgot this is used elsewhere.  added new consts `LATEST_STABLE_X_VALUE` instead of remaing the old one to avoid conflicts with existing code.